app/pods/agenda/agendaitems/route.js

```diff
@@ -1,57 +1,80 @@
 import Route from '@ember/routing/route';
 import { hash } from 'rsvp';
-import { isEmpty } from '@ember/utils';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
+import CONFIG from 'frontend-kaleidos/utils/config';
+import search from 'frontend-kaleidos/utils/mu-search';
 
-export default class AgendaItemsAgendaRoute extends Route {
+export default class AgendaAgendaitemsRoute extends Route {
   queryParams = {
     filter: {
-      refreshModel: false,
+      refreshModel: true,
+    },
+    showModifiedOnly: {
+      refreshModel: true,
+      as: 'toon_enkel_gewijzigd',
     },
   };
 
   @service sessionService;
   @service agendaService;
 
-  async model() {
+  async model(params, transition) {
     const {
       agenda,
+      meeting,
     } = this.modelFor('agenda');
-    const agendaitems = await this.store.query('agendaitem', {
+    console.log(params);
+    if (transition.from) {
+      console.log('From transition', transition.from);
+    }
+    let agendaitems = await this.store.query('agendaitem', {
       'filter[agenda][:id:]': agenda.id,
-      include: 'mandatees',
+      include: [
+        'mandatees'
+      ].join(','),
+      'fields[mandatees]': [
+        'title', // Display group header per agendaitems group
+        'priority' // Sorting agendaitems on minister protocol order
+      ].join(','),
+      'page[size]': CONFIG.MAX_PAGE_SIZE.AGENDAITEMS,
+      sort: 'show-as-remark,priority',
     });
 
-    const announcements = agendaitems.filter((agendaitem) => agendaitem.showAsRemark);
+    if (params.showModifiedOnly) {
+      // agendaService.agendaWithChanges is called by parent route
+      const matchingIds = this.agendaService.addedAgendaitems;
+      agendaitems = agendaitems.filter((ai) => matchingIds.includes(ai.id));
+    }
```

Comment:
@MikiDi Hier zit het probleem dat documenten niet aanzien worden al changes
hier wordt enkel gecheckt of het agendapunt nieuw is.
In het agendaitem model is er een property 'hasChanges'. Kunnen we die hier gebruiken ?

Reply:
Dank voor je analyse. Ik moet met de informatie uit https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/778#issuecomment-802915649 nog éen en ander in praktijk zetten. Zeker voor de toegevoegde/verwijderde punten zal een model refresh idd aan de orde zijn. `hasChanges` toevoegen op agendapunt lijkt me niet waar we naartoe willen. `hasChanges` is immers niet echt een eigenschap van het punt. Het hangt er van af in welke context (op welke agenda) je het punt bekijkt of het nu al dan niet `changed` is. Ik pas het aan.